sass/directives/_headers.scss

```diff
@@ -129,34 +104,36 @@
     p {
       max-width: 28em;
       margin-bottom: 0;
-      font-size: $hero-text-size;
+      font-size: strip-units($hero-text-size) * 1rem;
       color: $hero-text-color;
-
-      @media only screen and (max-width: $small-screen-max) {
-        display: none;
-      }
     }
   }
 }
 
-@mixin header-hero($background-image-large, $background-image-medium: nil) {
-  position: relative;
-  min-height: $header-height-large;
-  overflow: hidden;
+@mixin header-hero-product($background-image-large, $background-image-medium: nil) {
+  min-height: $header-height-product;
   background-color: $base-background-color;
   background-image: url(asset_path(#{$background-image-large}));
-  background-size: auto $header-height-large;
+  background-size: auto $header-height-product;
   background-position: right bottom;
   background-repeat: repeat no-repeat;
+  @include header-hero;
```

Comment:
move the @include up to line 114. the reason is that if the include has anything you you want to override you'll only be able to override it if it comes first

sometimes I'll put includes for things that just have prefixing in the spot where i'd put the un-prefixed property (i.e. if i'm using `@include display(flex)` it doesn't matter what order it's in) but if I'm including something like header-hero that comes with a bunch of stuff then I put it at the top